seleniumByTechnocredits/src/seleniumPractise/vishal/AlertHandling.java

```diff
@@ -0,0 +1,136 @@
+package seleniumPractise.vishal;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+import util.PropFileOperation;
+
+public class AlertHandling extends MasterClass {
+	WebDriver driver = Init.initChromeDriver();
+
+	public void verifyAlertDemo() throws IOException, InterruptedException {
+
+		Properties prop = PropFileOperation.loadProp();
+
+		String firstName = prop.getProperty("firstname");
+		String lastName = prop.getProperty("lastname");
+		String company = prop.getProperty("company");
+		String url = prop.getProperty("url");
+		// WebDriver driver = Init.initChromeDriver();
+
+		driver.get(url);
+		driver.findElement(By.linkText("Basic Elements")).click();
+		System.out.println("On Basic Element Page");
+		Thread.sleep(3000);
+		driver.findElement(By.xpath(prop.getProperty("firstnameXpath"))).sendKeys(firstName);
+		driver.findElement(By.xpath("//input[@name='ulname']")).sendKeys(lastName);
+		driver.findElement(By.xpath("//input[@name='cmpname']")).sendKeys(company);
+		// driver.findElement(By.xpath("//div[@name='secondSegment'][1]//button[text()='Submit']")).click();
+		click(driver, "//div[@name='secondSegment'][1]//button[text()='Submit']");
+		Thread.sleep(2000);
+		Alert alert = driver.switchTo().alert();
+
+		String expectedText = firstName + " and " + lastName + " and " + company;
```

Comment:
Here you can directly write like  String firstName = prop.getProperty("firstname");
No need to create a extra variable.